packages/deployer/subtasks/print-info.js

```diff
@@ -23,7 +23,10 @@ async function _printTitle() {
     logger.log(chalk.red(text));
   }
 
-  await printFiglet(readPackageJson().name);
+  try {
+    await printFiglet(readPackageJson().name);
+  } catch (err) {} // eslint-disable-line no-empty
```

Comment:
This is so that the root project doesn't require the user to have a `package.json`